plugin/src/main/java/net/aufdemrand/denizen/objects/dWorld.java

```diff
@@ -889,6 +889,62 @@ public String run(Attribute attribute, dObject object) {
             }
         });
 
+        // <--[tag]
+        // @attribute <w@world.list_gamerules>
+        // @returns dList(Element)
```

Comment:
Doc wrong

Reply:
Not sure what you think is wrong about this.
Is "dList(Element)" not correct since it returns a list of strings?

Reply:
I need a more specific hint than that because there are atleast 10 places in the tag documents that also has "dList(Element)" as a return

Reply:
There are zero such places.